python/lsst/analysis/tools/interfaces/datastore/_sasquatchDatastore.py

```diff
@@ -101,7 +114,17 @@ def __init__(
 
         self.accessToken = self.config.get("accessToken", "na")
 
-        self.namespace = self.config.get("namespace", "lsst.dm")
+        default_namespace = os.environ.get("DAF_BUTLER_SASQUATCH_NAMESPACE", "lsst.dm")
+        self.namespace = self.config.get("namespace", default_namespace)
```

Comment:
Isn't this overriding backwards?  I thought we want `self.config["namespace"]` unless `DAF_BUTLER_SASQUATCH_NAMESPACE` is defined, in which case we take it.

Reply:
Hmm. Right. Environment variable should always override. I will redo this.

Reply:
I think the same for the extras, for consistency, even if that one might go either way.

Reply:
Yes. Fixed.